react/features/recording/middleware.js

```diff
@@ -44,6 +44,8 @@ import {
     RECORDING_ON_SOUND_FILE
 } from './sounds';
 
+declare var config: Object;
```

Comment:
Please don't use the config global, access the state on the config feature.

Reply:
done